plsync/mlabzone.header.in

```diff
@@ -0,0 +1,52 @@
+;
+; Primary DNS zone file
+; NOTE: this file was automatically generated!!!
+; NOTE: DO NOT EDIT
+;
+
+$ORIGIN %(domain)s.
+$TTL    %(ttl)s
+
+; @	    IN 	    SOA	    alfred.cs.princeton.edu. support.planet-lab.org. (
+; @       IN      NS      alfred.cs.princeton.edu.
+; @       IN      NS      sns-pb.isc.org.
+
```

Comment:
The above three lines are commented out, and likely older, superseded values.  Rather than propagating them to the new configs, should we just remove them to make things cleaner and less confusing?